oozie-to-airflow/utils/file_archive_extractors.py

```diff
@@ -57,8 +57,8 @@ class FileExtractor:
     """ Extracts all file paths from an Oozie node """
 
     def __init__(self, oozie_node: Element, params: Dict[str, str]):
-        self.files: str = ""
-        self.hdfs_files: str = ""
+        self.files: List[str] = []
+        self.hdfs_files: List[str] = []
```

Comment:
We agreed that every PR should be justified. Introducing this change as a separate PR will not be justified. We agreed that commits we use as a history of changes and are not used to divide big changes into smaller ones.

Reply:
> We agreed that commits we use as a history of changes and are not used to divide big changes into smaller ones.
I don't recall this discussion. I think the division of PRs into commits should serve first the reviewers, to make code reviews easier, and secondly should make the history of changes clear.
If someone looks at the history of these lines and sees "Add Spark Mapper" it will not be a good explanation of what happened here.

But most importantly: **what about other mappers which rely in these variables being strings**?

Reply:
Other mappers had to be updated to be compatible with the new signature of the method.